### ISSUE TITLE
ZIPs starting with zero omit first character due to storing as an INT

### ISSUE THREAD

**Issue by wwwroth** [millbj92/US-Zip-Codes-JSON#12]
I noticed what I think is a bug. Storing ZIPs as INT will omit leading zeros. Here's an example of a ZIP in NJ:

```json
{
      "zip_code": 7837,
      "latitude": 41.12831,
      "longitude": -74.678956,
      "city": "Glasser",
      "state": "NJ",
      "county": "Sussex"
    }
```

This causes data loss since the first character defines the geographical region for the ZIP.

![image](https://github.com/millbj92/US-Zip-Codes-JSON/assets/3310210/19b1ce93-a0f6-48f2-aa5b-d58796590384)

Would you accept a PR to convert the data type to a STRING and correct all zip codes that begin with zero?

**Comment by millbj92** [comment]
Yeah, I think converting the zip to a string is acceptable. That's probably how it should have been done from the start. Good catch! I'll definitely accept a PR for that.
